The dev guide is very nice - thank you for writing that up!
https://docs.bokeh.org/en/latest/docs/dev_guide.html

The instructions how to set BOKEH_RESOURCES many options are provided without guidance which one to pick to be able to run the tests.

I chose the first `export BOKEH_RESOURCES=absolute-dev` but then run into this test collection RuntimeError from `tests/unit/bokeh/test_resources.py`.

When I try other options I get many unit tests to run, but still dozens of fails (and it's very slow).

Can you please improve the description here how to set it?


```
(bkdev) bokeh $ pytest tests/unit         
=============================================================================================== test session starts ================================================================================================
platform darwin -- Python 3.12.5, pytest-7.4.4, pluggy-1.5.0
rootdir: /Users/cdeil/code/oss/bokeh
configfile: pyproject.toml
plugins: cov-5.0.0, html-4.1.1, metadata-3.1.1, asyncio-0.21.1, anyio-4.4.0, timeout-2.3.1, xdist-3.6.1
asyncio: mode=Mode.STRICT
timeout: 90.0s
timeout method: signal
timeout func_only: False
collected 11255 items / 1 error                                                                                                                                                                                    

Bokeh selenium tests using 'chrome' driver (no-console-error assertions: ON)

====================================================================================================== ERRORS ======================================================================================================
_______________________________________________________________________________ ERROR collecting tests/unit/bokeh/test_resources.py ________________________________________________________________________________
tests/unit/bokeh/test_resources.py:45: in <module>
    raise RuntimeError("Cannot run the unit tests with BOKEH_RESOURCES set")
E   RuntimeError: Cannot run the unit tests with BOKEH_RESOURCES set
============================================================================================= short test summary info ==============================================================================================
ERROR tests/unit/bokeh/test_resources.py - RuntimeError: Cannot run the unit tests with BOKEH_RESOURCES set
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================================= 1 error in 3.85s =================================================================================================
```

--- a/docs/bokeh/source/docs/dev_guide/setup.rst
+++ b/docs/bokeh/source/docs/dev_guide/setup.rst
@@ -283,6 +283,10 @@ You will not see any effects of your local changes to BokehJS unless you
 configure Bokeh to use your local version of BokehJS instead of the default
 version from the CDN.
 
+Note that ``BOKEH_RESOURCES`` should only be set when running examples.
+When you run tests or build the docs, you should not set this variable
+(or unset it if it is already set) or you might get an error.
+
 You have the following three options to use your local version of BokehJS:
 
 Use ``absolute-dev``

--- a/docs/bokeh/source/docs/dev_guide/testing.rst
+++ b/docs/bokeh/source/docs/dev_guide/testing.rst
@@ -155,24 +155,23 @@ Unit tests
 
 Code coverage (Python unit tests)
     To create a coverage report for Python unit tests, use ``pytest`` with the
-    command-line options ``--cov=bokeh`` and ``--cov-config=tests/.coveragerc``:
+    command-line option ``--cov=bokeh``:
 
     .. code-block:: sh
 
-        pytest --cov=bokeh --cov-config=tests/.coveragerc
+        pytest --cov=bokeh
 
     Coverage with Bokeh's Python unit tests should be around 90%. Coverage
     reports are only relevant for Python unit tests. There are no coverage
     reports for other Python tests or for any of the JavaScript code of BokehJS.
 
-    You also have the option to add
-    ``--cov=bokeh --cov-config=tests/.coveragerc`` when running a specific
+    You also have the option to add ``--cov=bokeh`` when running a specific
     subset of Python unit tests. This adds a coverage report to the test
     results. For example:
 
     .. code-block:: sh
 
-        pytest --cov=bokeh --cov-config=tests/.coveragerc -m "not selenium" tests/unit/bokeh/test_objects.py
+        pytest --cov=bokeh --cov-report=html -m "not selenium" tests/unit/bokeh/test_objects.py
 
     .. seealso::
         Coverage reports use the pytest plugin `pytest-cov`_. For more

--- a/docs/bokeh/source/docs/dev_guide/testing.rst
+++ b/docs/bokeh/source/docs/dev_guide/testing.rst
@@ -124,7 +124,7 @@ when working with Bokeh's pytest-based tests:
 * ``-v``: Run test with more verbose output.
 * ``--driver``: Use a specific web driver for Selenium-based tests
   (``"chrome"``, ``"firefox"``, or ``"safari"``). For example:
-  ``pytest --driver="firefox" tests/integration/``.
+  ``pytest --driver="firefox" tests/unit/``.
 * ``--no-js``: Skip any JavaScript code and only test Python code.
 
 See the `pytest documentation`_ for more options.
@@ -180,14 +180,8 @@ Code coverage (Python unit tests)
 
 .. _contributor_guide_testing_local_python_integration:
 
-Integration tests
-    The Python-focused integration tests that simulate UI interactions
-    are currently removed until a more maintainable way is found to run
-    them via `Chrome`_ or `Chromium`_ and `Selenium`_ with the
-    `ChromeDriver`_ web driver.
-
 Cross integration tests
-    This is a variant of integration tests where on Bokeh's side a Python
+    There are some Python to JS interface tests where on Bokeh's side a Python
     code sample (a test case) is run, which produces JSON output with the
     serialized document. That JSON is then stored in the repository under
     ``tests/baselines/cross``. When adding a new test case, run:

